src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/IsInRangeConverter.shared.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class IsInRangeConverter : BindableObject, IValueConverter
```

Comment:
I am not sure if it's a good idea to make Converters extending bindableObjects

CC @jfversluis ?

Reply:
If it's a good idea, then probably we should make ValueConverterExtension extending BindableObject. And this class should be inherited from ValueConverterExtension

Reply:
@AndreiMisiukevich do you prefer static values?

Reply:
Hi @AndreiMisiukevich, who can help us to decide if we go with BindableObject or actually extend ValueConverterExtension?

Reply:
@pictos @AndreiMisiukevich I have made the changes based on the feedback, could you please let me know what you think?

Reply:
@GUOLDEV sorry for the late reply, rush week. I'll re-review ASAP.  I did a quick look and looks good.

Reply:
Thanks @pictos for the reply, I am in the same situation, having a busy week, I am taking baby steps, but I will get there 😅👍

Reply:
I have made ValueConverterExtension inherited from BindableObject.